src/components/signup/signup.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.css";
+import "../signup/signup.css";
+import logo1 from './logo1.png'
+
+
+const SignUp = () => {
+  return (
+      <div class="container">
+        <div class="signup-container">
+          <div class="signup-header">
+            <img class="logo" src={logo1} alt="Logo" />
+          </div>
+            <h2>Signup</h2>
```

Comment:
Is this how they often write it?  Do other sites write `Sign Up` , or `Sign up` or just `Signup`. Try to emulate the best practice around the text that is visible to the user.